### PR TITLE
Add more virtual return fixes.

### PR DESCRIPTION
**Pull request by tbaker57**
More fixes for the December Beginner sprint. With @abednarik 's help we might finish the sprint before Christmas :smile:

### CODE REVIEW

**Comment by cachedout** [comment]
Really nice! I like the way you've phrased all the error messages here. They are very clean and concise.

**Comment by tbaker57** [comment]
I'll fix up those lint errors ....

**Comment by abednarik** [comment]
Wow! Many thanks for that! Please is there are lint error, let me know. My
idea is to help you out, not to give you more work to do ;) Will check what
those lint errors to avoid having the same mistake.

BTW: In the list, there are lot of tasks that are already en develop branch
but documentation show that there are pending. Do you guys want to remove
that or search each commit and update the list?

Cheers.

2015-12-17 13:45 GMT-03:00 tbaker57 <notifications@github.com>:

**Comment by tbaker57** [comment]
I think the list on the wiki might not be complete (pinging @jfindlay for confirmation?). I'm using
```
find . -name "*.py" -exec grep -lPzo '(?s)def __virt((?!def).)*return ((?!def).)* False' {} \;
```
which finds (with a couple of false positives) all .py files with virtual return functions without a detailed error msg.

File: salt/modules/debian_service.py [hunk 1]
```diff
@@ -36,7 +36,8 @@ def __virtual__():
     '''
     if __grains__['os'] in ('Debian', 'Raspbian', 'Devuan') and not salt.utils.systemd.booted(__context__):
         return __virtualname__
-    return False
+    return (False, 'The debian_service module failed to load: only works on Debian, Raspbian or Devuan '
```
Comment (cachedout) [inline salt/modules/debian_service.py]: Please remove the trailing whitespace here.

**Comment by cachedout** [comment]
Looks like there may be duplicated effort, since this will no longer merge cleanly. Sorry to bother you, @tbaker57 but would you mind rebasing this so I can get it to merge? Thanks.

**Comment by tbaker57** [comment]
Whoah! Not sure where all those merge commits came from ? Happy to close this pull request now and start from scratch....

**Comment by cachedout** [comment]
Probably easiest to just open a new PR then. That's no problem.

**Pull request closed without merging**